service/src/test/java/datawave/microservice/audit/replay/AuditReplayTest.java

```diff
@@ -2725,24 +2698,24 @@ private static List<Status> toStatuses(String stringStatus) throws IOException {
         return statuses;
     }
     
-    private static Status toStatus(ResponseEntity<String> responseEntity) throws IOException {
+    private static Status toStatus(ResponseEntity<String> responseEntity) throws IOException, ParseException {
         return toStatus(responseEntity.getBody());
     }
     
     private static ObjectMapper objectMapper = new ObjectMapper();
     
-    private static Status toStatus(String stringStatus) throws IOException {
+    private static Status toStatus(String stringStatus) throws IOException, ParseException {
         Map<String,Object> map = objectMapper.readValue(stringStatus, new TypeReference<HashMap<String,Object>>() {});
         return mapToStatus(map);
     }
     
-    private static Status mapToStatus(Map<String,Object> map) {
+    private static Status mapToStatus(Map<String,Object> map) throws ParseException {
         Status status = new Status();
         status.setId((String) map.get("id"));
         status.setState(Status.ReplayState.valueOf((String) map.get("state")));
         status.setPathUri((String) map.get("pathUri"));
         status.setSendRate(Integer.toUnsignedLong((int) map.get("sendRate")));
-        status.setLastUpdated(ISODateTimeFormat.dateTime().parseDateTime((String) map.get("lastUpdated")).toDate());
+        status.setLastUpdated(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS").parse((String) map.get("lastUpdated")));
```

Comment:
Just a thought, no need to reply: I think java.time was meant to make joda obsolete.  SimpleDateFormat is dead to me so it catches my eye. https://stackoverflow.com/questions/38348938/equivalent-of-joda-isodatetimeformat-in-java-time

Reply:
Interesting.  I can update.